src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    public void setRadius(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public Figure draw() {
+
+        System.out.println("Figure: " + getClass().getSimpleName() + ", volume: "
+                + getVolume() + " sq.units, color: " + getColor() + ", radius: " + getRadius());
```

Comment:
```suggestion
                + getVolume() + " sq.units, color: " + getColor() + ", radius: " + radius);
```
you have direct access to fields in the class